freeplane_plugin_script/scripts/apiGenerator.groovy

```diff
@@ -323,16 +389,52 @@ makeApi(proxy, Proxy.DependencyLookup.class)
 makeApi(proxy, Dependencies.class)
 makeApi(proxy, ScriptUtils.class)
 
+makeApi(proxy, PhysicalUnit.class)
+makeApi(proxy, Dependencies.Element.class)
+makeApi(proxy, ConversionException.class)
+makeApi(proxy, NodeNotFoundException.class)
+makeApi(proxy, AttributeCondition.class)
+makeApi(proxy, AttributeValueSerializer.class)
+makeApi(proxy, Border.class)
+makeApi(proxy, ChildNodesAlignment.class)
+makeApi(proxy, ChildNodesLayout.class)
+makeApi(proxy, ChildrenSides.class)
+makeApi(proxy, ConditionalStyle.class)
+makeApi(proxy, ConditionalStyleNotFoundException.class)
+makeApi(proxy, ConditionalStyles.class)
+makeApi(proxy, Dash.class)
+makeApi(proxy, org.freeplane.api.FreeplaneVersion.class)
+makeApi(proxy, HeadlessLoader.class)
+makeApi(proxy, HeadlessMapCreator.class)
+makeApi(proxy, HorizontalTextAlignment.class)
+makeApi(proxy, LayoutOrientation.class)
+makeApi(proxy, NodeToComparableMapper.class)
+makeApi(proxy, TextWritingDirection.class)
+proxy.sortChildrenBy{it.plainText}
+
 def utils = createChild(newMap.root, UTILITES_NODE, null)
 initHeading(utils)
+// org.freeplane.plugin.script
 makeApi(utils, FreeplaneScriptBaseClass.class)
+//org.freeplane.core.ui.components
 makeApi(utils, UITools.class)
+makeApi(utils, UITools.Defaults.class)
+makeApi(utils, UITools.InsertEolAction.class)
+//org.freeplane.core.util
 makeApi(utils, LogUtils.class)
 makeApi(utils, HtmlUtils.class)
+makeApi(utils, HtmlUtils.IndexPair.class)
 makeApi(utils, TextUtils.class)
 makeApi(utils, MenuUtils.class)
+makeApi(utils, MenuUtils.MenuEntry.class)
+makeApi(utils, MenuUtils.MenuEntryTreeBuilder.class)
+// org.freeplane.plugin.script
 makeApi(utils, FreeplaneScriptBaseClass.ConfigProperties.class)
+//org.freeplane.core.util
 makeApi(utils, FreeplaneVersion.class)
+//org.freeplane.launcher
```

Comment:
Are you sure we can add the Launcher? Why do you think we should?

Reply:
I have no opinion on this.
I just added the classes that are shown in the API documentation. For example Lancher:

https://docs.freeplane.org/api/org/freeplane/launcher/Launcher.html

Reply:
Thanks, got it.